service/src/io/pedestal/http.clj

```diff
@@ -255,6 +255,7 @@
                                           (csrf/anti-forgery enable-csrf)])
                true (conj (middlewares/content-type {:mime-types ext-mime-types}))
                true (conj route/query-params)
+               true (conj route/path-params)
```

Comment:
Since this is a breaking change and has a slight performance impact, we should setup an option to turn this off -- by default it will be turned on.